py2pack/__init__.py

```diff
@@ -97,13 +97,16 @@ def _get_template_dirs():
     """existing directories where to search for jinja2 templates. The order
     is important. The first found template from the first found dir wins!"""
     return filter(lambda x: os.path.exists(x), [
+        os.path.join(i, "templates") for i in (
         # user dir
-        os.path.join(os.path.expanduser('~'), '.py2pack', 'templates'),
-        # system wide dir
-        os.path.join('/', 'usr', 'share', 'py2pack', 'templates'),
+        os.path.join(os.path.expanduser('~'), '.py2pack'),
+        platformdirs.user_data_dir(appname="py2pack"),
+        platformdirs.user_config_dir(appname="py2pack"),
         # usually inside the site-packages dir
-        os.path.join(os.path.dirname(os.path.abspath(__file__)), 'templates'),
-    ])
+        os.path.dirname(os.path.abspath(__file__)),
+        # system wide dir
+        *platformdirs.site_data_dir(appname="py2pack", multipath=True).split(":"),
```

Comment:
Looks almost good, but the [linter is complaining](https://github.com/openSUSE/py2pack/actions/runs/9182052259/job/25250152512?pr=199). If you could fix the flake8 complain, that could be nice.

The complain is about indentation in this block.